indico/MaKaC/webinterface/tpls/ConfDetailsFull.tpl

```diff
@@ -63,6 +63,8 @@
   </div>
   % endif
 
+  ${ template_hook('survey-announcement', event=conf) }
```

Comment:
But we do want that. That's the whole idea of a template hook - that modules/plugins can inject their own HTML at a specific place without the indico core having to know what it is.